Makefile.shared

```diff
@@ -103,7 +103,7 @@ CALC_VERSIONS=	\
 LINK_APP=	\
   ( $(SET_X);   \
     LIBDEPS="$${LIBDEPS:-$(LIBDEPS)}"; \
-    LDCMD="$${LDCMD:-$(CC)}"; LDFLAGS="$${LDFLAGS:-$(CFLAGS) $(LDFLAGS)}"; \
+    LDCMD="$${LDCMD:-$(CC)}"; LDFLAGS="$${LDFLAGS:-$(LDFLAGS)}"; \
```

Comment:
`$(CFLAGS)` should remain, there are flags for the linker that come that way, and I think both the compiler and the linker use the `-O` flags.